source/elements/oneDAL/source/introduction.rst

```diff
@@ -1,3 +1,20 @@
 ============
 Introduction
 ============
+
+oneDAL is library that provides building blocks covering all stages of data analytics: data acquisition
```

Comment:
use a legally correct name of the lib

Reply:
This is correct short version of library name

Reply:
This is the right way to use official names The **Intel® Data Analytics Acceleration Library (Intel® DAAL)** is the library of Intel® Architecture optimized building blocks covering all data analytics stages: data acquisition from a data source, preprocessing, transformation, data mining, modeling, validation, and decision

Reply:
these are the names of the classical DAAL you mentioned
for oneAPI: The approved usage is: oneAPI Data Analytics Library (oneDAL) (or Intel® oneAPI Data Analytics Library (oneDAL))

Reply:
I gave the analogy with the existent docs, and the q remains: what we should do in specs

Reply:
The names below are correct. The name that begins with Intel is the product name. THe name that begins with oneAPI is the spec name. Only use Intel is you are explicitly referring to the product and not the spec.

> for oneAPI: The approved usage is: oneAPI Data Analytics Library (oneDAL) (or Intel® oneAPI Data Analytics Library (oneDAL))

Reply:
I've change all mentions to use |dal_short_name| and |dal_full_name| aliases in RST. 
First mention on document  use both and then following mentions in document use just short